kube/resources/scheduler.yaml

```diff
@@ -81,8 +81,42 @@ spec:
                 configMapKeyRef:
                   name: airbyte-env
                   key: TEMPORAL_HOST
+            - name: TEMPORAL_WORKER_PORTS
+              valueFrom:
+                configMapKeyRef:
+                  name: airbyte-env
+                  key: TEMPORAL_WORKER_PORTS
           ports:
-            - containerPort: 8001
+            - containerPort: 9000
```

Comment:
@jrhizor  why is this necessary?

Reply:
This is the pool of worker ports that can be used by jobs on this scheduler node.